.github/workflows/run-tests.yml

```diff
@@ -0,0 +1,36 @@
+name: Run Tests
+
+on:
+  push:
+    branches: [ "main" ]
+  pull_request:
+    branches: [ "main" ]
+
+permissions:
+  contents: read
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
```

Comment:
all the steps except for "make test" were from GitHub's recommended PHP Composer action